apps/software/lib/software/controller/file_module.ex

```diff
@@ -29,11 +31,11 @@ defmodule Helix.Software.Controller.FileModule do
     end)
   end
 
-  @spec find(File.t) :: module_roles
-  def find(file) do
+  @spec get_file_modules(File.t | File.id) :: module_roles
+  def get_file_modules(file) do
     file
     |> FileModule.Query.from_file()
-    |> FileModule.Query.select_module_role_id_and_module_version()
+    |> select([m], {m.module_role_id, m.module_version})
```

Comment:
the table/record/schema name (`ModuleRole` and `FileModule`) and the field name (used interchangeably, they are `role`, `module` and `module_role`)